.vscode/launch.json

```diff
@@ -8,7 +8,14 @@
             "name": "support_sphere",
             "cwd": "src/support_sphere",
             "request": "launch",
-            "type": "dart"
+            "type": "dart",
+            // See: ../deployment/values.dev.yaml or the cloud yaml
```

Comment:
```suggestion
            // See: ../deployment/values.dev.yaml or the cloud yaml
            // Retrieve the `SUPABASE_ANON_KEY` from the `secret.jwt.anonKey` value in that file
            // Retrieve the `SUPABASE_URL` from the `studio.environment.SUPABASE_PUBLIC_URL` value in that file
            // This will allow the values to be passed into the frontend app during build.
```